cleanlab/models/keras.py

```diff
@@ -87,8 +89,15 @@ def get_params(self, deep=True):
             "model": self.model,
             "model_kwargs": self.model_kwargs,
             "compile_kwargs": self.compile_kwargs,
+            "params": self.params,
         }
 
+    def set_params(self, **params):
+        """Set the parameters of the Keras model."""
+        for key, value in params.items():
+            self.params[key] = value
```

Comment:
What's wrong with `update`?

```suggestion
        self.params.update(params)
```

Reply:
Nothing wrong! Just completely slipped my mind that this method exists 😶‍🌫️